packages/core/src/enums/Events.ts

```diff
@@ -211,6 +211,11 @@ enum Events {
    */
   VOLUME_SCROLL_OUT_OF_BOUNDS = 'CORNERSTONE_VOLUME_SCROLL_OUT_OF_BOUNDS',
 
+  /**
+   * Triggers when the scroll function is called on a volume.
+   */
+  VOLUME_SCROLL = 'CORNERSTONE_VOLUME_SCROLL',
```

Comment:
I noticed that we have STACK_VIEWPROT_SCROLL, so to maintain consistency, could you please think about adding VOLUME_VIEWPORT_SCROLL? I understand that we need to modify VOLUME_SCROLL_OUT_OF_BOUNDS, but that change might cause compatibility issues. I plan to make the update in version 2.x.

Reply:
Ok I'll do this!